jekyll/_cci2/caching.md

```diff
@@ -152,7 +150,7 @@ commands:
 ```
 {% endraw %}
 
-2) Use custom command in build to generate the concatenated `package-lock` file
+2) Use custom command in build to generate the concatenated `package-lock` file:
```

Comment:
Could you change the formatting here so they are proper numbered steps? `1.` etc.

Reply:
As the list is broken up by the code block, the numbering automatically restarts on the second point. Unlike in Asciidoc, there seems not to be a way of specifying a starting number. The only solution I've found is to indent the code block by four spaces - is that the preferred method? Or have I completely missed something simpler?!

Reply:
Have fixed with indents.